win32ss/user/ntuser/timer.c

```diff
@@ -219,8 +221,12 @@ IntSetTimer( PWND Window,
   {
       IntLockWindowlessTimerBitmap();
 
-      IDEvent = RtlFindClearBitsAndSet(&WindowLessTimersBitMap, 1, HintIndex);
-
+      IDEvent = RtlFindClearBitsAndSet(&WindowLessTimersBitMap, 1, HintIndex++);
+      if (IDEvent == (UINT_PTR) -1)
+	  {
+		  HintIndex = HINTINDEX_BEGIN_VALUE;
+		  IDEvent = RtlFindClearBitsAndSet(&WindowLessTimersBitMap, 1, HintIndex++);
```

Comment:
:+1: for handling the possible wrapping.

Reply:
@HBelusca, Why would this 2nd call succeed?